lib/fluent/plugin/out_newrelic.rb

```diff
@@ -50,6 +50,8 @@ def multi_workers_ready?
 
       def configure(conf)
         super
+
+        @license_key ||= ENV["NEW_RELIC_LICENSE_KEY"]
```

Comment:
Could you please also add support for the API key being passed via environment variables?

```suggestion
        @api_key ||= ENV["NEW_RELIC_API_KEY"]
        @license_key ||= ENV["NEW_RELIC_LICENSE_KEY"]
```